fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -989,12 +1011,7 @@ private CompletableFuture<Integer> moveDocsFromPartition(@Nonnull final LuceneRe
             return CompletableFuture.completedFuture(0);
         }
         RepartitionTimings timings = new RepartitionTimings();
-        final StoreTimerSnapshot timerSnapshot;
-        if (LOGGER.isDebugEnabled() && state.context.getTimer() != null) {
-            timerSnapshot = StoreTimerSnapshot.from(state.context.getTimer());
-        } else {
-            timerSnapshot = null;
-        }
+        final StoreTimerSnapshot timerSnapshot = getStoreTimerSnapshot();
```

Comment:
I think we should mark this teamscale warning as tolerated.
The line at which we call `getStoreTimerSnapshot` is important, and calling it later would not give us timing information.